### PR TITLE
Clawdit

### PR DESCRIPTION
**Pull request by dannylamb**
**Github Issue**: (https://github.com/Islandora/documentation/issues/1267)

# What does this Pull Request do?

Overhaul's README to meet our template, general de-CLAWing, and some better form text in a few spots that I touched up as I went along.

# How should this be tested?

Give the README a good once over.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/8-x-committers

### CODE REVIEW

**Comment by seth-shaw-unlv** [comment]
The tech call URL should be `https://github.com/Islandora/documentation/wiki#islandora-8-tech-calls`.

File: modules/islandora_text_extraction/README.md [hunk 1]
```diff
@@ -1,44 +1,49 @@
-# islandora_text_extraction
-### Connects Islandora 8 to Hypercube microservice and extracts text from PDFs
+# Islandora Text Extraction `
```
Comment (seth-shaw-unlv) [inline modules/islandora_text_extraction/README.md]: Back-tick shouldn't be here.

Comment (dannylamb) [reply modules/islandora_text_extraction/README.md]: I've got more submodules to go.... I'll take care of it

Comment (seth-shaw-unlv) [reply modules/islandora_text_extraction/README.md]: Ok. I'll just hold off until I hear from you that it is ready to go.

**Comment by dannylamb** [comment]
@seth-shaw-unlv Ok, I've gone through all the submodules now.

File: modules/islandora_breadcrumbs/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Islandora Audio
```
Comment (seth-shaw-unlv) [inline modules/islandora_breadcrumbs/README.md]: Copy/pasta error.

File: modules/islandora_core_feature/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Core configuration required by Islandora.
+
+## Requirements
+
+- `islandora`
+
+## Installation
+
+For a full digital repository solution (including a Homarus microservice), see our [installation documentation](https://islandora.github.io/documentation/installation/).
```
Comment (seth-shaw-unlv) [inline modules/islandora_core_feature/README.md]: Probably don't need the homarus reference here.

File: modules/islandora_iiif/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+Provides IIIF manifests using views.
+
+## Requirements
+
+- `islandora` and `islandora_core_feature`
+- A IIIF image server (such as Cantaloupe) 
+
+## Installation
+
+For a full digital repository solution (including a Homarus microservice), see our [installation documentation](https://islandora.github.io/documentation/installation/).
```
Comment (seth-shaw-unlv) [inline modules/islandora_iiif/README.md]: Homarus, again.

**Comment by seth-shaw-unlv** [comment]
@dannylamb, just those few little things I mentioned above, plus islandora_text_extraction is missing it's CONTRIBUTING.md.

**Comment by dannylamb** [comment]
@seth-shaw-unlv Kk, done.

**Comment by seth-shaw-unlv** [comment]
@dannylamb, coding standards.

**Comment by dannylamb** [comment]
:sob:

**Comment by dannylamb** [comment]
Alright, `phpcs` is satisfied on my box.  Hopefully nothing new pops on Travis.

**Pull request merged**